scripts/use-yarn-please.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const { spawnSync } = require('child_process');
+const npmPath = process.env.npm_execpath;
+
+const Strings = {
+  useYarnInstead: `Looks like you are trying to run "npm install", this repository has migrated to use Yarn as its package manager for installations.
+  `,
+  yarnDetected: `Great! Looks like you already have yarn installed!
+  `,
+  installYarn: `You current do not have an installation of Yarn in your PATH. Be sure to install the latest stable version of Yarn here:
+
+Download an installer here: https://yarnpkg.com/en/docs/install
+
+Windows users can run the installer here:
+
+  https://yarnpkg.com/latest.msi
+
+Mac users can run a one-liner for the install:
+
+  curl -o- -L https://yarnpkg.com/install.sh | bash
+`,
+  gettingStartedWithYarn: `To install UI Fabric monorepo dependencies and establish links between projects, simply run:
```

Comment:
Is there any equivalent of `rush unlink`?

Reply:
rush unlink would be `rm -rf node_modules` at the root or `git clean -fdx` for nuking. I don't think I've ever really used that though... you could do `yarn --force` to force installation of things.

Reply:
My main reason for using `rush unlink` is that IIRC `git clean -fdx` doesn't like symlinks. Or I occasionally used it for cleanup after I'd been manually messing with deps.

Reply:
Yup, in that case, you would just delete node_modules at the root. rush makes symlinks in many directories, so it would need that. It isn't something that is really needed in yarn.

Reply:
Yeah, with Rush you have to do `rush unlink` because otherwise `git clean -fdx` crawls into the source of local packages through the symlinks and deletes controlled code. So yarn doesn't use symlinks at all?

Reply:
I think yarn must use symlinking somehow. I'm trying your branch (after a full clean and remove of previous rush symlinks), did a build and then a `git clean-fdx` and am seeing a lot of deleted source:

![image](https://user-images.githubusercontent.com/26070760/61498740-53963280-a979-11e9-8248-028843b7c648.png)

Similar to rush, it seems like there'll need to be some unlinking before doing a git clean.